nx.json

```diff
@@ -32,7 +32,7 @@
     },
     "start": {
       "cache": true,
-      "dependsOn": ["^typecheck","^build"]
+      "dependsOn": ["^build"]
```

Comment:
removing systematic pre-requested typecheck as some projects are already doing it in background (twenty-front especially)